examples/hello_world/client/android/java/com/google/oak/hello_world/MainActivity.java

```diff
@@ -131,11 +135,23 @@ public void run() {
     /** Creates a TLS channel. */
     private ManagedChannel createChannel(String address) throws Exception {
       return OkHttpChannelBuilder.forTarget(address)
+          .intercept(MetadataUtils.newAttachHeadersInterceptor(getPublicUntrustedLabelMetadata()))
           .useTransportSecurity()
           .sslSocketFactory(getSocketFactory())
           .build();
     }
 
+    /** Gets the metadata associated with the empty label (public/untrusted). */
+    private Metadata getPublicUntrustedLabelMetadata() {
+      // TODO(#1066): Use a more restrictive Label.
+      Metadata result = new Metadata();
+      Label label = Label.newBuilder().build();
```

Comment:
Is it a public label?

Reply:
Yes, a label with no secrecy tags is public.